cicecore/cicedynB/dynamics/ice_dyn_evp.F90

```diff
@@ -1160,16 +1180,16 @@ subroutine evp (dt)
 
                uvel(:,:,:) = uvel(:,:,:)*uvm(:,:,:)
                vvel(:,:,:) = vvel(:,:,:)*uvm(:,:,:)
-
-            endif   ! grid_ice
-
-            ! U fields at NE corner
-            ! calls ice_haloUpdate, controls bundles and masks
-            call dyn_HaloUpdate (halo_info,          halo_info_mask,    &
+               ! U fields at NE corner
+               ! calls ice_haloUpdate, controls bundles and masks
+               call dyn_HaloUpdate (halo_info,          halo_info_mask,    &
                                  field_loc_NEcorner, field_type_vector, &
                                  uvel, vvel)
```

Comment:
same alignment fix needed here (I can't do a "Suggested change" since some lines are in the context).